### PR TITLE
Created UITest for Issue1758

### LINKED ISSUE #2238: Create UITest for Issue1758
This issue over a second changes the layout of a button. The final resting place of that layout could be checked in a UI Test

### PR DESCRIPTION
**Pull request by adrianknight89**
### Issues Resolved ### 

- fixes #2238

### PR Checklist ###

- [ ] Targets the correct branch
- [ ] Tests are passing (or failures are unrelated)

### CODE REVIEW

File: Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Issue1758.cs [hunk 1]
```diff
@@ -44,27 +56,55 @@ Layout Relative()
...
+
+			var oldRect = RunningApp.Query("Button").Single().Rect;
+
+			try
+			{
+				RunningApp.WaitForNoElement("Button", timeout: TimeSpan.FromSeconds(5));
+			}
+			catch
+			{
+				// swallow exception
```
Comment (jfversluis) [inline Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Issue1758.cs]: Why is this needed?

File: Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Issue1758.cs [hunk 2]
```diff
@@ -1,36 +1,48 @@
...
-		public Issue1758()
+
+		protected override void Init()
 		{
 			_list = new ListView { ItemsSource = new[] { "hello", "world", "from", "xamarin", "forms" } };
- 
-			_button = new Button { Text = "Button" };
- 
+
+			_button = new Button { Text = "Button", AutomationId = "Button" };
```
Comment (jfversluis) [inline Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Issue1758.cs]: To prevent magic string mistakes could you make the automation id a private const and use that?

File: Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Issue1758.cs (hunk 1 above)
Comment (jfversluis) [reply Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Issue1758.cs]: No, sure. Unfortunately, I don't think there is. But why is there an empty try/catch :)

Comment (adrianknight89) [reply Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Issue1758.cs]: Sorry. Catching only `TimeoutException` now.

**Comment by jfversluis** [comment]
Tests fail on this one both iOS and Android

image

image

> System.Exception : Error while performing WaitForNoElement(Marked("Button"), "Timed out waiting for no element...", 00:00:05, null, null)  ----> System.TimeoutException : Timed out waiting for no element...  at Xamarin.UITest.Utils.ErrorReporting.With (System.Action func, System.Object[] args, System.String memberName) [0x00052] in <a0c3f09cbf9049cbb8d3a680a53dcf46>:0   at Xamarin.UITest.Android.AndroidApp.WaitForNoElement (System.Func`2[T,TResult] query, System.String timeoutMessage, System.Nullable`1[T] timeout, System.Nullable`1[T] retryFrequency, System.Nullable`1[T] postTimeout) [0x00032] in <a0c3f09cbf9049cbb8d3a680a53dcf46>:0   at Xamarin.UITest.Android.AndroidApp.WaitForNoElement (System.String marked, System.String timeoutMessage, System.Nullable`1[T] timeout, System.Nullable`1[T] retryFrequency, System.Nullable`1[T] postTimeout) [0x0000d] in <a0c3f09cbf9049cbb8d3a680a53dcf46>:0   at Xamarin.Forms.Controls.ScreenshotConditionalApp.WaitForNoElement (System.String marked, System.String timeoutMessage, System.Nullable`1[T] timeout, System.Nullable`1[T] retryFrequency, System.Nullable`1[T] postTimeout) [0x00001] in <c60ce05d69f1440d8f21a1fe9797aff4>:0   at Xamarin.Forms.Controls.Issues.Issue1758.CheckButtonPosition () [0x00071] in <c60ce05d69f1440d8f21a1fe9797aff4>:0   at (wrapper managed-to-native) System.Reflection.MonoMethod:InternalInvoke (System.Reflection.MonoMethod,object,object[],System.Exception&)  at System.Reflection.MonoMethod.Invoke (System.Object obj, System.Reflection.BindingFlags invokeAttr, System.Reflection.Binder binder, System.Object[] parameters, System.Globalization.CultureInfo culture) [0x00032] in <48b95f3df5804531818f80e28ec60191>:0 --TimeoutException  at Xamarin.UITest.Shared.WaitForHelper.WaitFor (System.Func`1[TResult] predicate, System.String timeoutMessage, System.Nullable`1[T] timeout, System.Nullable`1[T] retryFrequency, System.Nullable`1[T] postTimeout) [0x0002a] in <a0c3f09cbf9049cbb8d3a680a53dcf46>:0   at Xamarin.UITest.Android.AndroidApp+<>c__DisplayClass81_0.<WaitForNoElement>b__0 () [0x00054] in <a0c3f09cbf9049cbb8d3a680a53dcf46>:0   at Xamarin.UITest.Utils.ErrorReporting.With (System.Action func, System.Object[] args, System.String memberName) [0x00006] in <a0c3f09cbf9049cbb8d3a680a53dcf46>:0

**Comment by adrianknight89** [comment]
I don't have time to fix this anymore. Maybe somebody else can take a look at it.
